app/models/incoming_links_report.rb

```diff
@@ -27,6 +28,7 @@ def self.find(type, _opts = {})
     report = IncomingLinksReport.new(type)
 
     report.start_date = _opts[:start_date] || 30.days.ago
+    report.end_date = _opts[:end_date] || Time.now.utc.end_of_day
```

Comment:
Is there a reason we use `Time.now.utc` instead of following the server's timezone?